### DISCUSSION TITLE
Per-User Conf

### DISCUSSION THREAD

**Discussion by JohnFawkes** [Lissy93/dashy#240] [Help and Support] [answered]
Is there any way to do per user confs? For example, one user has certain sections on the dashboard when they login and another user has other sections? Is that possible to do?

**Comment by JohnFawkes** [comment, marked as answer]
Nevermind. I found what i am looking for in the closed issues https://github.com/Lissy93/dashy/issues/165

**Reply by lissy93** [reply]
To help anyone looking for this in the fututre, this is outlined in the [Granular Access](https://github.com/Lissy93/dashy/blob/master/docs/authentication.md#granular-access) docs :)

**Reply by JohnFawkes** [reply]
I actually think this should either be added to the demo conf.yml or the docs. I never would of found this if i didnt look through the closed issues. I may make a PR for it to be added to the documentation later today. I downloaded the demo config to get started and based my setup off that

**Reply by JohnFawkes** [reply]
> To help anyone looking for this in the fututre, this is outlined in the [Granular Access](https://github.com/Lissy93/dashy/blob/master/docs/authentication.md#granular-access) docs :)

OMG! I skipped over it and never even saw it..... Thanks!
